modules/node/nodes.tf

```diff
@@ -4,15 +4,18 @@ resource "scaleway_ip" "node_ip" {
 }
 
 # scaleway instance
-resource "scaleway_server" "instance" {
+resource "scaleway_instance_server" "instance" {
 
+  zone           = "${var.scaleway_zone}"
   count          = "${var.instance_count}"
   name           = "${terraform.workspace}-manager-${count.index + 1}"
   image          = "${data.scaleway_image.docker.id}"
   type           = "${var.manager_instance_type}"
   security_group = "${scaleway_instance_security_group.node_rules.id}"
   public_ip      = "${element(scaleway_ip.swarm_manager_ip.*.ip, count.index)}"
   tags           = var.tags
+  policy_type    = "max_availability"
```

Comment:
`placement_group_id`, the placement group policy must be set in a `scaleway_instance_placement_group` resource